src/Nancy.Validation.FluentValidation/FluentValidationValidatorFactory.cs

```diff
@@ -45,9 +45,19 @@ private IValidator GetValidatorInstance(Type type)
         {
             var fullType =
                 CreateValidatorType(type);
+            var available = this.validators
+                .Where(validator => fullType.GetTypeInfo().IsAssignableFrom(validator.GetType()))
+                .ToArray();
+            if (available.Length > 1)
+            {
+                var names = string.Join(", ", available.Select(v => v.GetType().Name));
+                var message =
+                    $"Ambiguous choice between multiple validators for type {type.Name}. "

```

Comment:
We have not yet started using C# 6 features, please use a `string.Concat` instead